docker/api/requirements.txt

```diff
@@ -3,7 +3,7 @@ cloudevents==1.2.0
 fastapi[all]==0.68.1
 fastapi-pagination==0.9.3
 passlib==1.7.4
-pydantic==1.8.2
+pydantic==1.10.5
```

Comment:
I think this change should be part of the commit https://github.com/kernelci/kernelci-api/pull/213/commits/10edc60edf086587a4a676487d8f05a7471f6488 that uses `FileUrl`. Otherwise, the commit can result in a broken codebase.

Reply:
The commit to upgrade Pydantic is before the code changes that depend on it.

Reply:
Oh, sorry. I didn't check the order properly.